JDK11 support: jaxb

jaxb has been removed from the JDK and is now available as separate maven dependency. Compiling with openjdk11 fails with:

```
[ERROR] heritrix3/engine/src/main/java/org/archive/crawler/restlet/XmlMarshaller.java:[34,33] package javax.xml.bind.annotation does not exist
[ERROR] heritrix3/engine/src/main/java/org/archive/crawler/restlet/XmlMarshaller.java:[35,33] package javax.xml.bind.annotation does not exist
[ERROR] heritrix3/engine/src/main/java/org/archive/crawler/restlet/XmlMarshaller.java:[36,33] package javax.xml.bind.annotation does not exist
[ERROR] heritrix3/engine/src/main/java/org/archive/crawler/restlet/models/ScriptModel.java:[9,33] package javax.xml.bind.annotation does not exist
[ERROR] heritrix3/engine/src/main/java/org/archive/crawler/restlet/models/ScriptModel.java:[10,33] package javax.xml.bind.annotation does not exist
[ERROR] heritrix3/engine/src/main/java/org/archive/crawler/restlet/models/ScriptModel.java:[14,2] cannot find symbol
  symbol: class XmlRootElement
[ERROR] heritrix3/engine/src/main/java/org/archive/crawler/restlet/models/ScriptModel.java:[15,2] cannot find symbol
  symbol: class XmlType
[ERROR] heritrix3/engine/src/main/java/org/archive/crawler/restlet/XmlMarshaller.java:[141,9] cannot find symbol
  symbol:   class XmlRootElement
  location: class org.archive.crawler.restlet.XmlMarshaller
[ERROR] heritrix3/engine/src/main/java/org/archive/crawler/restlet/XmlMarshaller.java:[141,59] cannot find symbol
  symbol:   class XmlRootElement
  location: class org.archive.crawler.restlet.XmlMarshaller
[ERROR] heritrix3/engine/src/main/java/org/archive/crawler/restlet/XmlMarshaller.java:[160,13] cannot find symbol
  symbol:   class XmlType
  location: class org.archive.crawler.restlet.XmlMarshaller
[ERROR] heritrix3/engine/src/main/java/org/archive/crawler/restlet/XmlMarshaller.java:[160,60] cannot find symbol
  symbol:   class XmlType
  location: class org.archive.crawler.restlet.XmlMarshaller
[ERROR] heritrix3/engine/src/main/java/org/archive/crawler/restlet/XmlMarshaller.java:[170,50] cannot find symbol
  symbol:   class XmlTransient
  location: class org.archive.crawler.restlet.XmlMarshaller
```

--- a/engine/pom.xml
+++ b/engine/pom.xml
@@ -69,6 +69,18 @@
 			<artifactId>joda-time</artifactId>
 			<version>1.6</version>
 		</dependency>
+		<!-- jaxb is no longer included in jdk11+ -->
+		<dependency>
+			<groupId>javax.xml.bind</groupId>
+			<artifactId>jaxb-api</artifactId>
+			<version>2.3.1</version>
+		</dependency>
+		<dependency>
+			<groupId>org.glassfish.jaxb</groupId>
+			<artifactId>jaxb-runtime</artifactId>
+			<version>2.3.1</version>
+			<scope>runtime</scope>
+		</dependency>
 		<dependency>
 			<groupId>org.littleshoot</groupId>
 			<artifactId>littleproxy</artifactId>

